src/core/base_mixin/game_flow_mixin.py

```
@@ -162,19 +157,25 @@ def skip_dialog(self):
                 self.log_info("skip_dialog 超时退出")
                 return False
             if self.find_one("skip_dialog_esc", horizontal_variance=0.05):
-                self.press_esc()
-                self.sleep(0.1)
+                self.log_info("检测到可跳过对话，发送 ESC")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)